### DISCUSSION TITLE
The class BusinessEntity is not properly integrated

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1393] [Business Entities (BE)] [answered]
The class https://spec.edmcouncil.org/fibo/ontology/BE/LegalEntities/LegalPersons/BusinessEntity has only 2 subclasses (https://spec.edmcouncil.org/fibo/ontology/BE/SoleProprietorships/SoleProprietorships/SoleProprietorship and https://spec.edmcouncil.org/fibo/ontology/BE/PrivateLimitedCompanies/PrivateLimitedCompanies/PrivateCompanyWithLimitedLiability) which do not include Corporation and other formal organizations.

Further, there seems to be no relationship between it and https://spec.edmcouncil.org/fibo/ontology/BE/FunctionalEntities/FunctionalEntities/FunctionalBusinessEntity

**Comment by ElisaKendall** [comment, marked as answer]
In addition to better integrating this ontology, I think it makes sense to merge SoleProprietorships with LegalPersons, given that it is small and unlikely to be further extended.  Sole proprietorships are simpler entities than LLCs, for example, which are a kind of PrivateLimitedCompany.  The latter may be expanded with additional details, such as relationships with partnerships in some cases, where sole proprietorships is quite small and unlikely to need additional details based on the use cases that we currently have or are aware of.

**Comment by rivettp** [comment]
It's not technically correct to say that a LLC is a kind of PrivateLimitedCompany.
The differences vary by jurisdiction, but see for example https://www.upcounsel.com/difference-between-llc-and-private-limited-company .
Even that article is not complete with respect to taxation (which I know is out of scope for FIBO) since a LLC can choose to be taxed as a S Corporation.

**Comment by ElisaKendall** [comment]
I consulted with a business attorney and another business auditor / consultant responsible for creating/filling business entities.  An LLC is disjoint with an S Corporation.  There is no such thing as a private limited company in the US, per se.  The US equivalent is an LLC.  Both an LLC and S-Corporation are pass-thru entities from a tax perspective.  Corporations file taxes using form 1120, 1120S for an S-Corporation, form 1065 for a partnership, and form 1065 for an LLC.  The shareholder taxes for S-corporations, partnerships and LLCs are filed on a schedule K or K-1.

**Comment by ElisaKendall** [comment]
We can confirm that again with WF attorneys who assisted in development of this part of the model in the first place, though.

A business entity is an independent party, which is why you don't see the relationship there to the functional entity.  It is a parent to a sole proprietorship as well as other kinds of business entities, so perhaps we should make it the parent of others.  But with respect to corporation, it would be a for-profit corporation that would be a child of business entity, and possiblly some not-for-profit corporations, but htat would have to be determined on a case by case basis.
